<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VLHT10O" failures="1" tests="1" name="twoTest" time="14.973" errors="0" timestamp="2024-04-09T09:42:17 PDT">
  <testcase classname="Tests.TestTwo" name="two" time="12.159">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.TestTwo.two(TestTwo.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- two -->
</testsuite> <!-- twoTest -->
